model/k8s/metrics.yaml

```diff
@@ -63,6 +63,34 @@ groups:
       - ref: network.interface.name
       - ref: network.io.direction
 
+  # k8s.container.* metrics
+  - id: metric.k8s.container.status.state
```

Comment:
please add these metric definitions to k8s metrics markdown file - https://github.com/open-telemetry/semantic-conventions/blob/main/docs/system/k8s-metrics.md